turbinia/workers/plaso.py

```diff
@@ -114,7 +126,7 @@ def run(self, evidence, result):
         return result
 
     cmd.extend(['--logfile', plaso_log])
-    cmd.extend([plaso_file, evidence.device_path])
+    cmd.extend([plaso_file, evidence.source_path])
```

Comment:
IIRC, I don't think `source_path` is what we want here since after some of the preprocessors will run and create loopback devices.  Did you run into a problem with using `device_path`?  Note that right now in HEAD some of the pre-processors run when they don't necessarily need to, so can cause some extra failures.    There's a few things I need to do, but https://github.com/google/turbinia/pull/485 should fix that part.

Reply:
Actually, I think source_path is the correct thing here (see comments in https://github.com/google/turbinia/issues/389), but it looks like the pre-processor isn't setting this correctly.

@rgayon Should we be updating the `source_path` at the end of the end of the preprocessors here and in other evidence types?  https://github.com/google/turbinia/blob/0af1ff19836998ed73657225248c451d9551881d/turbinia/evidence.py#L390

Reply:
I think I had to use `source_path` since I was running tests out of a directory and not a device.